docs/vc-http-api.yml

```diff
@@ -41,6 +41,28 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/status:
+    post:
+      tags:
+        - Issuer
+      summary: Updates the status of an issued credential
+      operationId: updateCredentialStatus
+      description: Updates the status of an issued credential.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateCredentialStatus"
+        description: Parameters for updating the status of the issued credential.
+      responses:
+        "200":
+          description: Credential status successfully updated!
+        "400":
+          description: Bad Request!
+        "404":
+          description: Credential not found!
+        "500":
+          description: Internal Server Error!
```

Comment:
No exclamation marks in error conditions! :P 

Also, these descriptions are not very descriptive. We need to fix this (not now, but later)

Reply:
But errors are exceptional and hence should be emphasised :) will update